CONTRIBUTING.md

```diff
@@ -40,10 +40,20 @@ source .tox/dev/bin/activate
 tox -e fmt           # update your code according to linting rules
 tox -e lint          # code style
 tox -e unit          # unit tests
-tox -e integration   # integration tests
 tox                  # runs 'fmt', 'lint', and 'unit' environments
 ```
 
+It is recommended to use Multipass VM for running integration tests in a predictable environment.
+
+```
+multipass launch --cpus 4 --memory 8G --disk 50G --name charm-dev charm-dev
+multipass stop
```

Comment:
question: forgive my ignorance, I am not experienced with Multipass
From the [doc](https://multipass.run/docs/stop-command#:~:text=The%20multipass%20stop%20command%20without,Running%20status%20can%20be%20stopped.) I see that the `stop` command will stop any instances (if present). So from this I understand that stop command will stop the instance launched with the `launch` command.
Why do we need to run `multipass stop` ?

Reply:
Unfortunately, `multipass mount` with `--type=native` doesn't work when the VM is running.

But thank you for the highlight. I spotted that I forgot to add the VM name to `multipass stop` command. Fixing that and adding an explicit `multipass start`. The `multipass shell` runs the VM implicitly, but additional step will make it more more obvious.

Reply:
Amazing, that's more than I could ask for. Thank you!